content/clubs/clubs.json

```diff
@@ -0,0 +1,42 @@
+{
+    "clubsData": [
+        {
+            "name": "UTM MCSS",
+            "desc": "The MCSS encourages a social, academic and career-related support system among students and faculty. The Society encourages faculty and student interaction outside of the formal lecture, tutorial and lab setting.",
+            "imgSrc": "/logos/mcss.png",
+            "link": "https://www.facebook.com/utmmcss",
+            "button": "Check out our Facebook page!"
+        },
+        {
+            "name": "UTM Robotics Club",
+            "desc": "With a new Robotics faculty at UTM, the Robotics club seeks to empower the club's members and the broader student community to collaborate effectively on robotics projects. We hope to create a place for innovation for applied computer science, and promote interest in robotics and its applications for students at UTM.",
+            "imgSrc": "/logos/robotics.png",
+            "link": "https://utmrobotics.com",
+            "button": "Check out our website!",
+            "reversed": true
```

Comment:
I don't think the reversed logic should be here. You could iterate over the indices in the template and reverse it for every even index